fake_service/schema/services.py

```diff
@@ -1,64 +1,35 @@
 import csv
-from .models import Schema, Dataset
+from schema.models import Schema, Dataset
 from django.core.files import File
 import os
-from .utils import make_generetions, generate_cache_task_key
+from schema.utils import generate_fake_data, generate_cache_task_key
 from typing import Dict, List, Any
 
 
 class FakeDataWrite:
 
-    def _write_data_and_save_to_file(self, count: str, schema: Dict, file_name: str):
+    def _write_data_and_save_to_file(self, row_nums: str, schema: Dict, file_name: str):
         column_separator = schema.column_separator
         string_character = schema.string_character
-        to_csv = make_generetions(data=schema.fields, count=count)
-        keys = to_csv[0].keys()
+        data_to_csv = generate_fake_data(data=schema.fields, row_nums=row_nums)
```

Comment:
I dont think its a good idea to call this function here. You have service named FakeDataWrite(bad naming as well, i think would be better FE `CSVDataWriterService`) That should be responsible for creating csv file and store this file somewhere locally. But you generating data inside this service. So You should split responsibility between this 2 functionalities. I think you should have another service(class or function, it doesnt really matter). And inside this function(or class): 1) Generate fake data
2) use CSV Writer to write this data to file